R/model_epidemic_default.R

```diff
@@ -0,0 +1,45 @@
+
+#' SIR-V epidemic model.
+#'
+#' @param t The times.
+#' @param init The initial conditions.
+#' @param params The parameters.
+#'
+#' @return A list with numeric elements, giving the status of each compartment.
+#' @keywords internal
+#'
+epidemic_default <- function(t, init, params) {
```

Comment:
```suggestion
epidemic_default <- function(t, y, params) {
```

I don't recommend using `init` for this variable name and it will not be the initial state after the first iteration of `lsoda()` and this creates confusion.

Reply:
Accepting this suggestion for now; this function will be removed in upcoming PRs.